src/Sulu/Bundle/MediaBundle/Entity/CollectionRepository.php

```diff
@@ -339,6 +342,8 @@ private function getIdsQuery(
             }
         }
 
+        $queryBuilder->addOrderBy('collection.id', 'ASC');
```

Comment:
1. Is sorting by the id really necessary?
2. If yes, it should probably only be done if no other order arguments are applied.